sdk/src/types/block/slot/epoch.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::{Deref, Display, From, FromStr};
+
+/// Timeline is divided into slots, and each epoch has a corresponding epoch index.
```

Comment:
This reads a little bit weird tbh, not really defining Epoch.
Can't we say that timeline is divided into epochs, themselves divided into slots?